eox_core/edxapp_wrapper/backends/users_m_v1_test.py

```diff
@@ -0,0 +1,102 @@
+"""
+Test backend to get CourseEnrollment Model.
+"""
+
+from django.contrib.auth.models import Permission
+
+USERNAME_MAX_LENGTH = 30
+
+
+def get_edxapp_user(**kwargs):
+    """
+    Return a fake user
+    """
+    return object
```

Comment:
I suggest using something like [Mock](https://docs.python.org/3/library/unittest.mock.html#the-mock-class) instead of object:
```suggestion
    return Mock()
```
A Mock is an object you can use for tests. The mock objects allow you to call it, know if you are doing the right calls, and are flexible enough to modify it if needed. 

Let me know your thoughts.

Other comments:
- [x] Drop circleci
- [x] Requirements updated
- [x] Drop support to Django22 and add Python3.10
- [x] Added workflow to publish in Pypi
- [x] README updated
- [x] Github test passed 

The rest is okay for me 😊

Reply:
Thanks for your comment @MaferMazu, I use your suggestion